content/03.methods.md

```diff
@@ -19,5 +19,20 @@
 ### Gene Expression Abundance Estimation
 
 ### RNA Fusion Calling and Prioritization
+#### Gene Fusion Detection
+We set up [Arriba v1.1.0](https://github.com/suhrig/arriba/) and STAR-Fusion 1.5.0 [@doi:10.1101/120295] fusion detection tools using CWL on CAVATICA.
+For both these tools we used aligned BAM and chimeric SAM files from STAR as inputs and GRCh38_gencode_v27 GTF for gene annotation.
```

Comment:
```suggestion
We input aligned BAM and chimeric SAM files from STAR to both of these methods and used GRCh38_gencode_v27 GTF for gene annotation.
```
Do you want to link GRCh38_gencode_v27's GTF file here? Since we can use hyperlinks, it seems like we might as well.